platform/platform-resources/src/META-INF/PlatformLangPlugin.xml

```diff
@@ -69,6 +69,7 @@
     <registryKey key="unknown.sdk.auto" defaultValue="true" description="Checks and resolves unknown SDKs automatically on start"/>
     <registryKey key="unknown.sdk.modal.jps" defaultValue="true" description="Run unknown JDK test before JPS build is started"/>
     <registryKey key="unknown.sdk.show.editor.actions" defaultValue="true" description="Show editor suggestions to fix missing SKDs"/>
+    <registryKey key="kotlin.not.configured.show.notification" defaultValue="true" description="Show notification about Kotlin missing configuration"/>
```

Comment:
Since the new key is Kotlin specific, I would suggest moving this registry key into a Kotlin xml file.
I think the best might be `jvm-k1.xml` as it contains the registration for the `KotlinSetupEnvironmentNotificationProvider` class as well.